old_syntax.js

```diff
@@ -0,0 +1,12 @@
+
+    <!-- old links -->
+    <!-- semantic css -->
```

Comment:
Expected an identifier and instead saw '<'.<br>Expected an operator and instead saw '!'.<br>Expected an operator and instead saw '--'.<br>Unrecoverable syntax error. (23% scanned).